src/terms_s00.jl

```diff
@@ -0,0 +1,112 @@
+const s00p = (
+
+# 1-6
+    94.00e-6,
+  3808.35e-6,
+  -119.94e-6,
+-72574.09e-6,
+    27.70e-6,
+    15.61e-6
+)
+
+const s000 = (
+
+# 1-10
+(( 0,  0,  0,  0,  1,  0,  0,  0), -2640.73e-6,   0.39e-6 ),
```

Comment:
I would prefer if you split this into two separate tuples of tuples.

Reply:
(( 0,  0,  0,  0,  1,  0,  0,  0), (-2640.73e-6,   0.39e-6 )) like this?

Reply:
```julia
const s000_1 = (
    ( 0,  0,  0,  0,  1,  0,  0,  0),
    ...
)

const s000_2 = (
    (-2640.73e-6,   0.39e-6),
    ...
)
```

or something like it.

Reply:
I am not sure whether it is a problem or not but your data structure would make iteration type-unstable.

```julia
a = s000[1]

typeof(a[1]) # NTuple{8,Int64}
typeof(a[2]) # Float64
typeof(a[3]) # Float64 

for x in a
    # x is type-unstable and performance will suffer
end
```